### ISSUE TITLE
[FIX] Implicit call to super()

### ISSUE THREAD

**Issue by EclipsedButter** [Grimrukh/soulstruct-blender#69]
bug introduced with 2.1.1: `# ERROR: Cannot import FLVER: c2730. Error: super(type, obj): obj must be an instance or subtype of type`.
Seems to happen for any game, likely just introduced as a result of splitting FLVER into BaseFLVER, FLVER, and FLVER0. Untested with DeS. Current fix, use explicit call:
```
soulstruct/base/models/flver/core.py.FLVER.guess_rigged (line 473)
    has_bone_weights = super(FLVER, self).guess_rigged()
```

**Comment by Grimrukh** [comment]
Ah, my bad, fixing right now!

**Comment by Grimrukh** [comment]
Fixed this in v2.1.2, and other non-DeS-breaking bugs with DCX compression and FLVER/collision export.
